src/pages/accounts/AccountDetails.tsx

```diff
@@ -1,451 +1,321 @@
-                                    size='small'
-                                    startIcon={<FaPlus style={{ fill: '#3E79F7', width: '12px' }} />}
-                                    style={{ textTransform: 'capitalize', fontWeight: 600, fontSize: '16px' }}
-                                >
-                                    Add Attachments
-                                </Button>
-                                {/* </div> */}
-                            </div>
+                  <p>
+                    <strong>Industry</strong>
+                  </p>
+                  <p>{account.account_obj.industry}</p>
 
-                            <div style={{ padding: '10px 10px 10px 15px', marginTop: '5%' }}>
-                                {/* {lead && lead.lead_attachment} */}
-                                {accountDetails?.account_attachment?.length ? accountDetails?.account_attachment.map((pic: any, i: any) =>
-                                    <Box key={i} sx={{ width: '100px', height: '100px', border: '0.5px solid gray', borderRadius: '5px' }}>
-                                        <img src={pic} alt={pic} />
-                                    </Box>
-                                ) : ''}
-                            </div>
-                        </Box>
-                    </Box>
+                  <p>
+                    <strong>Assigned To</strong>
+                  </p>
+                  <p>
+                   {account?.account_obj?.assigned_to?.map((user, index) => (
+                    <React.Fragment key={index}>
+                    {user.user_details.email}
+                  <br />
+                    </React.Fragment>
+                     ))}
+                  </p>
+                  <p>
+                    <strong>Deals</strong>
+                  </p>
+                  {<p>
+                    {Array.isArray(account?.deals) ? (
+                      account.deals
+                      .filter((deal) => deal.account === accountId)
+                      .map((deal, index) => (
+                      <React.Fragment key={index}>
+                      {deal.name}
+                      <br />
+                      </React.Fragment>
+                      ))
+                      ) : (
+                      'No deals available'
+                      )}
+                  </p>
+                  }
                 </Box>
-            </div>
+              </Grid>
+
+              <Grid item xs={12} sm={4}>
+                <Grid container mb={5}>
+                  <Box height={300} borderColor={'black'}>
+                    <Grid item>
+                      <ButtonsComponent />
+                    </Grid>
+                  </Box>
+                </Grid>
+                <Grid container>
+                  <Grid item paddingRight={10}>
+                    <DescriptionComponent
+                      initialDescription={account.description}
```

Comment:
`account.account_obj.description`

Reply:
Thank you for the review. I have changed the code related to description section.
I will merge now.